google_guest_agent/metadata.go

```diff
@@ -242,11 +243,12 @@ func getMetadata(ctx context.Context, hang bool) (*metadata, error) {
 	}
 	finalURL += ("&last_etag=" + etag)
 
-	req, err := http.NewRequestWithContext(ctx, "GET", finalURL, nil)
+	req, err := http.NewRequest("GET", finalURL, nil)
```

Comment:
Why the switch back from WithContext?

Reply:
not supported in the go version we use in builds, apparently.